README.md

```diff
@@ -1 +1,23 @@
-# developer-documentation
\ No newline at end of file
+# eZ Platform Developer Documentation
+
+This repository is the source for the developer documentation for eZ Platform,
+an open source CMS based on the Symfony Full Stack Framework in PHP.
+
+# Resources
+
+1. eZ Platform Developer Hub: https://ezplatform.com/
+1. eZ Platform Repository: https://github.com/ezsystems/ezplatform
+1. eZ Platform Requirements: https://github.com/ezsystems/requirements
+1. Open JIRA board: https://jira.ez.no/
+1. eZ Systems Website: https://ez.no/
+1. Old docs site (no longer in use): https://doc.ez.no/display/MAIN/eZ+Documentation+Center
+
+## How to Contribute
+https://ez-systems-developer-documentation.readthedocs-hosted.com/en/master/community_resources/contributing/#contribute-to-documentation
```

Comment:
Is there a way to format a URL that is always pointing to the latest version, or will it get locked in to, say, something like http://doc.ezplatform.com/en/1.11/x/y/z ?